src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -170,99 +157,61 @@ def connect(self):
-
-            for target in [
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o1"]],
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o2"]],
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o3"]],
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o4"]],
-            ]:
-                self._set_device_parameter(target, "connect_out0", value="off")  # Default after reboot = True
-                self._set_device_parameter(target, "connect_out1", value="off")  # Default after reboot = True
-                self._set_device_parameter(target, "connect_out2", value="off")  # Default after reboot = True
-                self._set_device_parameter(target, "connect_out3", value="off")  # Default after reboot = True
-
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o1"]],
-                "connect_out0",
-                value="I",
-            )
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o2"]],
-                "connect_out1",
-                value="Q",
-            )
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o3"]],
-                "connect_out2",
-                value="I",
-            )
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o4"]],
-                "connect_out3",
-                value="Q",
-            )
+                target.set("cont_mode_en_awg_path0", False)  # Default after reboot = False
+                target.set("cont_mode_en_awg_path1", False)
+                target.set("cont_mode_waveform_idx_awg_path0", 0)
+                target.set("cont_mode_waveform_idx_awg_path1", 0)
+                target.set("marker_ovr_en", True)
+                target.set("marker_ovr_value", 15)
+                target.set("mixer_corr_gain_ratio", 1)
+                target.set("mixer_corr_phase_offset_degree", 0)
+                target.set("offset_awg_path0", 0)
+                target.set("offset_awg_path1", 0)
+                target.set("sync_en", False)
+                target.set("upsample_rate_awg_path0", 0)
+                target.set("upsample_rate_awg_path1", 0)
+                target.set("connect_out0", "off")  # Default after reboot = True
+                target.set("connect_out1", "off")
+                target.set("connect_out2", "off")
+                target.set("connect_out3", "off")
+
+            self.device.sequencers[self.DEFAULT_SEQUENCERS["o1"]].set("connect_out0", "I")
+            self.device.sequencers[self.DEFAULT_SEQUENCERS["o2"]].set("connect_out1", "Q")
+            self.device.sequencers[self.DEFAULT_SEQUENCERS["o3"]].set("connect_out2", "I")
+            self.device.sequencers[self.DEFAULT_SEQUENCERS["o4"]].set("connect_out3", "Q")
```

Comment:
Maybe here is better to keep it explicitly since it may change soon as follows:
``` python
            self.device.sequencers[self.DEFAULT_SEQUENCERS["o1"]].set("connect_out0", "I")
            self.device.sequencers[self.DEFAULT_SEQUENCERS["o2"]].set("connect_out1", "Q")
            self.device.sequencers[self.DEFAULT_SEQUENCERS["o3"]].set("connect_out2", "I")
            self.device.sequencers[self.DEFAULT_SEQUENCERS["o4"]].set("connect_out3", "I")
```
or some other combinations in which the loop won't be straightforward.

Reply:
The non-trivial information is the second argument of `.set()`, just make a list for them :)